.github/workflows/ci.yaml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Configure AWS creds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
```

Comment:
Any reason we've gone with 8 rather than 11?

Reply:
To keep it in line with our other projects. Also there is no current ARM AMI with cdk base for Java 11. When we address updating Java across our projects in a future health week, this should be an easy migration